subsys/bluetooth/mesh/vnd/Kconfig

```diff
@@ -50,6 +50,13 @@ config BT_MESH_SILVAIR_ENOCEAN_WAIT_TIME
 	  Time for which button must be held to start sending delta_set messages,
 	  in milliseconds.
 
+config BT_MESH_SILVAIR_ENOCEAN_PHASE_B_DELAY
```

Comment:
I don't have the spec in front of me, so I have no idea what this means, and the help text didn't help me understand what it does, to be honest :smile: 

Just from looking at the entry title and help text, it should be possible to determine what this is and how it affects the observed behavior. Also, is there any reason to configure it, and why does it default to 1090?

Reply:
You know, as the father of this abomination of a Kconfig option I could not agree more :P It is now removed completely.